### PR TITLE
Protobuf message fields moved to get_fields method

### PR DESCRIPTION
**Pull request by ph4r05**
Related to:
- https://github.com/trezor/trezor-common/issues/171
- https://github.com/trezor/trezor-common/pull/212
- 2 commits, version bump + re-generated messages and templates, protobuf.py update

### CODE REVIEW

File: src/protobuf.py [hunk 1]
```diff
@@ -204,8 +207,9 @@ def __init__(self):
         setattr(msg, fname, fvalue)
 
     # fill missing fields
-    for tag in msg.FIELDS:
-        field = msg.FIELDS[tag]
+    fields = msg.get_fields()
```
Comment (jpochyla) [inline src/protobuf.py]: Redundant?

Comment (jpochyla) [reply src/protobuf.py]: Nonono, I mean that `fields` already contains the message fields, since line 155.

Comment (ph4r05) [reply src/protobuf.py]: 😅 true

**Comment by jpochyla** [comment]
Thanks!

**Pull request merged**